### PR TITLE
DOCS-152 Update staking api flow docs

### PR DESCRIPTION
**Pull request by yash-sharma1**
This PR is to update the NEAR Staking API flows under `/guides/staking-api/Near` as a PoC for the pre-sales team following the ticket DOCS-152.

### CODE REVIEW

File: docs/guides/manage-and-secure-api-keys.mdx [hunk 1]
```diff
@@ -24,7 +24,8 @@ You can create multiple API keys for a DataHub app. You can view your app's API
 <img
   src={img4}
   alt="Dashboard API Key"
-  width="500"a
+  width="500"
+  a
```
Comment (paulpomerleau) [inline docs/guides/manage-and-secure-api-keys.mdx:28]: is the 'a' here on 28 a typo?

Comment (yash-sharma1) [reply docs/guides/manage-and-secure-api-keys.mdx:28]: must be typo, didn't notice that

Comment (yash-sharma1) [reply docs/guides/manage-and-secure-api-keys.mdx:28]: lol made a new commit to update it but no change

**Review by paulpomerleau — APPROVED** [review]
looks good, need a review on the copy and feedback for the improvement. but this is definitely an improvement. approved, let's keep iterating as feedback comes in :-)

**Pull request merged**